src/v2/ion/i18nTransformer.js

```diff
@@ -364,6 +364,12 @@ const getMessage = (message) => {
     let i18nKey = message.i18n.key;
     let i18nParams = message.i18n.params || [];
 
+    // Sometimes API sends params: [""] an array with empty string.
+    // example - error-authenticator-enroll-password-common mock
+    if (i18nParams.length === 1 && i18nParams[0] === '') {
```

Comment:
😞  This is unfortunate. Ideally, we should fix the API. Can we create a JIRA and link here ?

Reply:
https://oktainc.atlassian.net/browse/OKTA-398080